packages/fxa-graphql-api/src/test/lib/db/models/profile/avatar-provider.sql

```diff
@@ -0,0 +1,6 @@
+CREATE TABLE `avatar_providers` (
+  `id` int(11) NOT NULL AUTO_INCREMENT,
+  `name` varchar(32) COLLATE utf8_unicode_ci NOT NULL,
+  PRIMARY KEY (`id`),
+  UNIQUE KEY `avatar_providers_name` (`name`)
+) ENGINE=InnoDB AUTO_INCREMENT=3 DEFAULT CHARSET=utf8 COLLATE=utf8_unicode_ci
```

Comment:
It's shame to have to duplicate these, but I guess we can try to consolidate with the ones in auth-db-mysql over time?

Reply:
Indeed, we could throw them all into a single file at some point.